routes/weather.ts

```diff
@@ -5,7 +5,13 @@ import * as SunCalc from "suncalc";
 import * as moment from "moment-timezone";
 import * as geoTZ from "geo-tz";
 
-import { AdjustmentOptions, GeoCoordinates, TimeData, WateringData, WeatherData, WeatherProvider } from "../types";
+import { GeoCoordinates, TimeData, WateringData, WeatherData, WeatherProvider } from "../types";
```

Comment:
Would it make sense to have WateringData, WeatherData, WeatherProvider in a "./WeatherProvider" file like you use for Adjustment related types

Reply:
Yeah, I think I'll do that refactoring in a separate PR.

Reply:
Resolving in favor of #55.